packages/icons/README.md

```diff
@@ -16,6 +17,14 @@ npm install @vibe/icons
 import { Close } from "@vibe/icons";
 ```
 
+Alternatively, you can import the raw SVG files
+
+```javascript
+import { Close as CloseSvg } from "@vibe/icons/raw";
```

Comment:
I'm wondering if it should be the initial thing we said "raw" or just "svg"
@rivka-ungar @talkor wdyt?